### ISSUE TITLE
Getting "No actions detected" when there are in fact actions

### ISSUE THREAD

**Issue by dougpagani** [mheap/pin-github-action#8]
It seems to be broken -- maybe due to the github deprecation warning I see?

Here is the output:
![image](https://user-images.githubusercontent.com/18250753/101185963-85687180-3620-11eb-80c8-8a1b8356e1f5.png)

Here is the file it was run against:

```yaml
name: Test Workflow for Pinner
on:
  workflow_dispatch


jobs:
  log-event:
    timeout-minutes: 1
    runs-on: ubuntu-latest
    steps:
      - name: "log the runner's envars"
        run: |
          env
      - uses: actions/upload-artifact@v2
        name: upload artifact
        with:
          name: gha-${{ github.event_name }}.json
          path: ${{ github.event_path }}

      - name: Sticky Pull Request Comment
        uses: marocchino/sticky-pull-request-comment@v1.6.0
        with:
          path: pr-comment.txt
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

        uses: dawidd6/action-send-mail@v2
        with:
          server_address: smtp.gmail.com
          server_port: 465
          username: ${{secrets.MAIL_USERNAME}}
          password: ${{secrets.MAIL_PASSWORD}}
          subject: "subject"
          body: "body"
          to: fakeemail@gmail.com
          from: Workflow File
          content_type: text/plain




```

**Comment by dougpagani** [comment]
This project seems like a great idea for security! Would love to get it to work. I'd probably go through the trouble of adding an "all" option that iterated through the workflow files. It could be leveraged, then, as an easily npm-run-scriptable ops tool.

I would consider adding an example for local actions, too, of things to avoid pinning. (_i.e._ `uses: ./.github/actions/myAction`)

**Comment by mheap** [comment]
Thanks for raising this. I've fixed the bug and released it as `v1.1.2`

I've also captured the issue with local actions as #10 

Regarding the `all` option, if you think it's a big improvement on using a loop please submit an issue and it'll be considered as a feature enhancement e.g.

```bash
# With a loop
for i in `ls --color=never`; do
  pin-github-action $i;
done

# Detect directory and process all
pin-github-action .github/workflows
```
